packages/wrangler/src/pages/build.ts

```diff
@@ -105,6 +105,12 @@ export function Options(yargs: CommonYargsArgv) {
 				deprecated: true,
 				hidden: true,
 			},
+			external: {
+				describe: "A list of imports to exclude from bundling",
+				type: "string",
+				array: true,
+				hidden: true,
```

Comment:
Should we keep this hidden?

Reply:
Seems reasonable to expose it, unless we think we'll take it away.